apps/question-the-docs/backend/ai/utils/text.py

```diff
@@ -1,14 +1,13 @@
 "AI helper functions for text processing."
 
-import dataclasses as dc
+import enum
 
 import pandas as pd
 
 
-@dc.dataclass
-class TextProcessing:
-    stride: int = 5
-    window_size: int = 10
+class TextProcessing(enum.Enum):
```

Comment:
I don't believe this is an `Enum`? We are not selecting a single field, but rather require both fields? Perhaps I am wrong about this though.